roles/openshift_logging/templates/es.j2

```diff
@@ -95,6 +95,13 @@ spec:
               readOnly: true
             - name: elasticsearch-storage
               mountPath: /elasticsearch/persistent
+          readinessProbe:
+            exec:
+              command:
+              - "/usr/share/elasticsearch/probe/readiness.sh"
+            initialDelaySeconds: 5
+            timeoutSeconds: 5
```

Comment:
@rhcarvalho Interestingly the API you linked defines the default value for `timeoutSeconds` to be 10 seconds :-) But I think in practice it is 1 second as you say.

So, as of now, the only timeout that can lead to readiness probe failure is the one that is implemented internally within the check script itself. Probably it makes sense then to remove the `timeoutSeconds` settings from this PR.